src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -0,0 +1,22 @@
+package com.kodilla.testing.collection;
+
+import java.util.ArrayList;
+
+class OddNumbersExterminator {
+
+    public static ArrayList<Integer> exterminate(ArrayList<Integer> numbers) {
+
+        ArrayList<Integer> evenNumbers = new ArrayList<Integer>();
+
+        for (int i=0; i<numbers.size(); i++) {
+            if (numbers.get(i) % 2 == 0) {
+                evenNumbers.add(numbers.get(i));
+            }
+        }
+
+        return evenNumbers;
+    }
```

Comment:
to jest czesc innego zadania ktore powinno byc juz na masterze. Upewnij sie prosze ze zadanie z Exterminator jest zmergowane